Library/Formula/jmxtrans.rb

```diff
@@ -1,15 +1,27 @@
-require 'formula'
-
 class Jmxtrans < Formula
-  homepage 'https://github.com/jmxtrans/jmxtrans'
-  url 'https://github.com/downloads/jmxtrans/jmxtrans/jmxtrans-20121016-151320-36564abc7e.zip'
-  version '20121016'
-  sha1 '2b2dc4727de16a30e8b7ac84b9bb4cb7b1ea4a6f'
+  homepage "https://github.com/jmxtrans/jmxtrans"
+  url "https://github.com/jmxtrans/jmxtrans/archive/jmxtrans-249.tar.gz"
+  version "20150330"
```

Comment:
Maybe `20150330-249` or something? Would be good to get `249` in there somewhere.